pkg/mount/mount.go

```diff
@@ -479,8 +481,20 @@ func (m *Mounter) Mount(
 
 	// Record previous state of the path
 	pathWasReadOnly := m.isPathSetImmutable(path)
+	isBindMounted := false
+
 	if err := m.makeMountpathReadOnly(path); err != nil {
-		return fmt.Errorf("failed to make %s readonly Err: %v", path, err)
+		if strings.Contains(err.Error(), "Inappropriate ioctl for device") {
```

Comment:
when do we see this error? this is not expected in normal path. right?

Reply:
If the mountpath such as `/var/lib/kubelet` is on a filesystem like overlay which does not support chattr